### DISCUSSION TITLE
Feature Idea - Load the controlnet preprocessor and model only ONCE for batch img2img

### DISCUSSION THREAD

**Discussion by sahildeshpande** [Mikubill/sd-webui-controlnet#1136] [Ideas]
Currently, if you use batch img2img with CN, the model is preprocessed and loaded for each image which takes a lot of time. Is there a way to make this faster by loading the model just once ?

**Comment by ljleb** [comment]
Yes. You have to increase your models cache size to be >= to the number of models to be loaded simultaneously.
